### PR TITLE
complete task 8

### PR DESCRIPTION
**Pull request by winniekwoknorth**

### CODE REVIEW

File: __tests__/tests.test.js [hunk 1]
```diff
@@ -291,42 +291,77 @@ describe('post/api/articles/:article_id/comments', () => {
...
+    }
+    return request(app)
+      .patch('/api/articles/1/')
+      .send(update)
+      .expect(201)
+      .then((res) => {
+        expect(res.body.articles.votes).toBe(101) 
+      })
+  })
+  test.only('201: ignore properties add beside inc_votes and return a updated article', () => {
```
Comment (nc-hev-mag) [inline __tests__/tests.test.js]: remove .only from test

File: __tests__/tests.test.js [hunk 2]
```diff
@@ -291,42 +291,77 @@ describe('post/api/articles/:article_id/comments', () => {
...
-//       .then((res) => {
-//         console.log(res.body)
-//         expect(res.body.articles.votes).toBe(101) 
-//       })
-//   })
-// })
\ No newline at end of file
+// task 8
+describe('patch/api/articles/:article_id/', () => {
+  test.only('201: response with add comments to database', () => {
```
Comment (nc-hev-mag) [inline __tests__/tests.test.js]: remove .only from test

File: err.js [hunk 3]
```diff
@@ -16,5 +16,6 @@ exports.handlePsqlErrors = ((err, req, res, next) => {
     else next(err)
 })
 exports.handleServerErrors = ((err, req, res, next) => {
+    console.log(err)
```
Comment (nc-hev-mag) [inline err.js]: remove console.log when finished

**Review by nc-hev-mag — CHANGES_REQUESTED** [review]
Hi Winnie! Great stuff here so far. 💯 Still a couple of tests needed for this endpoint though:

Status 400: Bad request, sending invalid inc_votes value
E.g. inc_votes value is not a number.

Status 400: Bad request, article_id is not a number
E.g. patch(`/api/articles/not-a-number`)

Please resubmit once added! 😄

**Comment by winniekwoknorth** [comment]
update subject to comment

**Review by aloush-dev — APPROVED** [review]
Awesome great job implementing feedback!

Merge when ready.

**Pull request merged**